tests/FakeItEasy.Tests/Core/CallOrderingContextTests.cs

```diff
@@ -0,0 +1,146 @@
+﻿namespace FakeItEasy.Tests.Core

```

Comment:
> I'd hope that they never had to use anything but FakeItEasy, not that I'd
be against putting super advanced but publicly accessible features in
another namespace sometime…

Ditto.

BTW as I said, I've not had the time to follow this PR properly but you
guys seem to be doing a fantastic job. If, when you feel ready to merge,
you'd like me to do a once over, please feel free to ping me. Or at any
time in between. Not to suggest that I'm more of an authority, just if you
feel you need a fresh pair of eyes on it.
On 2 Mar 2016 12:35 pm, "Blair Conrad" <notifications@github.com> wrote:

> In tests/FakeItEasy.Tests/Core/CallOrderingContextTests.cs
> <https://github.com/FakeItEasy/FakeItEasy/pull/605#discussion_r54711913>:
>
> > @@ -0,0 +1,146 @@
> > +﻿namespace FakeItEasy.Tests.Core
>
> Probably?
> Sounds good. Me, I'd hope that they never had to use anything but
> FakeItEasy, not that I'd be against putting super advanced but publicly
> accessible features in another namespace sometime…
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FakeItEasy/FakeItEasy/pull/605/files#r54711913>.
>